src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -160,6 +161,9 @@ export interface IPlotsTabProps {
     store:ResultsViewPageStore;
 };
 
+export type Option = {value:string, label:string};
```

Comment:
Could conflict pretty quickly with other Option types. Maybe more specific name?

Reply:
Renamed to `PlotsTabOption`